packages/ra-ui-materialui/src/preferences/Configurable.stories.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from 'react';
+import {
+    PreferencesEditorContextProvider,
+    useSetInspectorTitle,
+    useStore,
+    I18nContextProvider,
+    memoryStore,
+    StoreContextProvider,
+    usePreferenceKey,
+} from 'ra-core';
+import { Box, Typography } from '@mui/material';
+import TimelineIcon from '@mui/icons-material/Timeline';
+
+import { InspectorButton } from './InspectorButton';
+import { Inspector } from './Inspector';
+import { Configurable } from './Configurable';
+
+export default {
+    title: 'ra-ui-materialui/preferences/Configurable',
+};
+
+const TextBlock = ({ children }: { children?: React.ReactNode }) => {
+    const preferenceKey = usePreferenceKey();
+    const [color] = useStore(`${preferenceKey}.color`, '#ffffff');
+    return (
+        <Box
+            border="solid 1px lightgrey"
+            borderRadius={3}
+            p={1}
+            width={300}
+            bgcolor={color}
+        >
+            <Typography variant="h6">Lorem ipsum</Typography>
+            <Typography>
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit. Qui,
+                quia rem? Nulla asperiores ea beatae iure, dignissimos ut
+                perferendis nemo reiciendis reprehenderit, consequuntur debitis
+                maiores! Quaerat dolor unde dolorum qui.
+            </Typography>
+            {children}
+        </Box>
+    );
+};
+
+const TextBlockEditor = () => {
```

Comment:
The editor doesn't use `usePreferenceInput`. It would be great to have the changes on enter